### DISCUSSION TITLE
Support for proxydhcp variables (disbaled by default)

### DISCUSSION THREAD

**Discussion by scott0bailey** [antonym/netboot.xyz#1339] [Q&A]
Hello. 
I found this > https://github.com/netbootxyz/netboot.xyz/pull/953

How do I enable this? Im running netbootxyz in docker.

My TFTP server is being sent via dnsmasq and having this problem:
https://discourse.linuxserver.io/t/netbootxyz-looking-at-wrong-ip-for-local-menu/3240

![image](https://github.com/netbootxyz/netboot.xyz/assets/37932761/9da2daa5-bbe7-4d28-b118-5c6aeb644918)

the #953 indicates its disabled by default.
Anyway to enable this?

**Comment by centerionware** [comment]
https://github.com/centerionware/dnsmasq-proxy
Just use a separate dnsmasq container to keep things seperate and simple. The included dnsmasq is operating as the tftp server not a dhcp server or dhcp proxy server.
